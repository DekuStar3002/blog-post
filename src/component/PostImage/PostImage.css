.img {
  height: 50%;
  width: 100%;
}

.img img {
  height: 100%;
  width: 100%;
}