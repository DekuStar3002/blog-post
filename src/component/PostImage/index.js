import React from 'react'
import './PostImage.css';
const PostImage = ({ imgSrc }) => {
  return (
    <div className='img'>
        <img src={imgSrc} alt='imgSrc'/>
    </div>
  )
}

export default PostImage;