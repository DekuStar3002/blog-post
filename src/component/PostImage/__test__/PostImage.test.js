import PostImage from '..';
import { render } from "@testing-library/react";

describe('PostImage', () => { 
  it('should render correctly', () => {
    const props = {
      imgSrc: 'imgSrc'
    }
    const { asFragment } = render(<PostImage {...props}/>);
    expect(asFragment()).toMatchSnapshot();
  })
});