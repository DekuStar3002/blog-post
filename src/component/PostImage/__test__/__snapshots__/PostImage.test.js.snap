// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostImage should render correctly 1`] = `
<DocumentFragment>
  <div
    class="img"
  >
    <img
      alt="imgSrc"
      src="imgSrc"
    />
  </div>
</DocumentFragment>
`;
